colors.js

```diff
@@ -0,0 +1,10 @@
+export const colors = {
+  // https://colorbrewer2.org/#type=qualitative&scheme=Set1&n=4
```

Comment:
These specific choices are probably not great. Blue routes look like rivers? Very happy for design guidance here. It may not be too important to get things exactly right for the 4 categories today, if we'll want to distinguish more stuff in the schema soon.

Also, all of these choices depend very much on the basemap. If we switched to a greyscale style, or satellite, all of this will need to change.

Reply:
Oh yeah, and the route snapper overloads red, blue, and black to mean something else. Conflicts with things here.

Reply:
> These specific choices are probably not great. Blue routes look like rivers? Very happy for design guidance here. It may not be too important to get things exactly right for the 4 categories today, if we'll want to distinguish more stuff in the schema soon.

I don't have any better colour suggestions at the moment...